### DISCUSSION TITLE
Unable to establish a connection to query-engine-node-api library

### DISCUSSION THREAD

**Discussion by jlorezz** [prisma/prisma#14072] [Q&A]
Currently getting: `Unable to establish a connection to query-engine-node-api library` when spinning up the app.

**Comment by jlorezz** [comment]
### Versions: 
`"prisma": "^3.15.2",`
`"@prisma/client": "^3.15.2",`

**Reply by janpio** [reply]
Can you provide more information? Usually there should be more information.
Can you upgrade to Prisma 4.0.0 which further improved that error message please?

**Reply by jlorezz** [reply]
@janpio #14073

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
